java21/src/test/java/org/jdbi/v3/core/statement/TestTelemetry.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.statement;
```

Comment:
this will be a problem when we really want to do java modules. This now overlaps with the core. 

We can absolutely *require* java 17 to build (and test) as long as we still compile to 11. From what I see, jfr should work on Java 16+, so 17 would be fine.

Reply:
I appreciate the problem but am not sure what to do about it at this time.
The other option I can see will be to make many more Jdbi types and methods public, which doesn't feel great.

Do you have a suggestion as to what we should do about this?

I'm OK with not worrying about it for now - this is only test code and we can change it at any time later with no compatibility issues.

Reply:
"not worry for now" :-)